src/evict.c

```diff
@@ -710,12 +712,12 @@ int performEvictions(void) {
     latencyEndMonitor(latency);
     latencyAddSampleIfNeeded("eviction-cycle",latency);
 
-exceeded_info:
+update_metrics:
     if (result == EVICT_RUNNING || result == EVICT_FAIL) {
         if (server.stat_last_eviction_exceeded_time == 0)
             elapsedStart(&server.stat_last_eviction_exceeded_time);
     } else if (server.stat_last_eviction_exceeded_time != 0) {
-        server.stat_total_eviction_exceeded_time += elapsedUs(server.stat_last_eviction_exceeded_time);
+        server.stat_total_eviction_exceeded_time += elapsedUs(server.stat_last_eviction_exceeded_time) / 1000;
```

Comment:
not sure if i'm contradicting something i said in the past (i don't think so), i'm not sure it's wise to do that division here.
we're summing very short intervals, we may be summing 0 all the time, or losing some precision.
i think it may be better to keep that variable in `us`, and do another division in INFO.

IIRC the wrap around is still non-realistic for micros in 64 bit.